src/miner.cpp

```diff
@@ -30,6 +31,10 @@
 #include <boost/tuple/tuple.hpp>
 #include <mutex>
 
+#ifndef USE_TROMP_EQUIHASH
+// #define USE_TROMP_EQUIHASH
+#endif
```

Comment:
We expect that many users will just be using pre-built binaries (like our Debian packages), so instead we should use a runtime flag to determine which solver is run (see example below).